ADO.NET/OAuthSampleApp/OAuthSampleApp.cs

```diff
@@ -3,14 +3,16 @@
 using Newtonsoft.Json;
 using System.Configuration;
 
-internal class AdoOauthSample
+internal class OAuthSampleApp
 {
     static void Main(string[] args)
     {
         VerticaConnectionStringBuilder vcsb = new VerticaConnectionStringBuilder();
         vcsb.ConnectionString = ConfigurationManager.ConnectionStrings["ConnectionString"].ConnectionString;
 
-      //  vcsb.OAuthJsonConfig = GetTokenRefreshJsonConfig();
+        // json config is required if doing token refresh
+        // comment out line below if only using access token
+        vcsb.OAuthJsonConfig = GetTokenRefreshJsonConfig();

```

Comment:
I've made this application taking refresh tokens into consideration. We could just scope the sample app to access tokens, but it wouldn't be that interesting. It would be the same as the app in GettingStarted with another parameter in the connection string...